Features/bootstrap/MainContext.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use QATools\BehatExtension\Context\QAToolsContext;
+
+class MainContext extends QAToolsContext
+{
+
+	/**
+	 * @Given I'm on ':page'
```

Comment:
According to http://docs.behat.org/en/latest/guides/2.definitions.html the single or double quotes are not required around parameter names. Parameters would be detected in scenario even if  somebody adds quotes by accident:

```
I'm on My Page
I'm on "My Page"
I'm on 'My Page'
```
Not sure how spaces are handled through.